spec/lucky_cli/generators/validators_spec.cr

```diff
@@ -0,0 +1,40 @@
+require "../../spec_helper"
+
+private def validate(string)
+  LuckyCli::Generators::Validators::ProjectName.valid? string
+end
+
+private def sanitize(string)
+  LuckyCli::Generators::Validators::ProjectName.sanitize string
+end
+
+
```

Comment:
Could you run `crystal tool format spec src`? I think that should clean up this extra newline